include/git2/blob.h

```diff
@@ -120,6 +120,7 @@ typedef enum {
  * The options used when applying filter options to a file.
```

Comment:
Can this have the blurb that's found in other structs like https://github.com/libgit2/libgit2/blob/fe41e5823986aff01054b7db4b869b2896ea4c30/include/git2/clone.h#L96-L104 to use the better-supported initialization patterns?

Reply:
Updated documentation in new commit.

> `git_blob_filter_options_init` still needed; should be added in another
commit/PR (it's out of the scope of this PR, #5759), update this
documentation again.

Reply:
I agree with @lhchavez, we should fix this while we're here.  `git_blob_filter_options_init` is in scope.  I pushed up a commit that added it.  Would you mind revising the doc block to refer to it?